Sources/NIOOpenSSL/SSLConnection.swift

```diff
@@ -49,22 +48,10 @@ internal final class SSLConnection {
         return false
     }
     
-    init? (_ ssl: UnsafeMutablePointer<SSL>, parentContext: SSLContext) {
+    init (_ ssl: UnsafeMutablePointer<SSL>, parentContext: SSLContext) {
         self.ssl = ssl
```

Comment:
don't you need to openssl_up_ref that?

Reply:
No, this function takes an owned ref. As it's purely internal, we can guarantee safe usage.